src/cljs/org/broadinstitute/firecloud_ui/common/entity_table.cljs

```diff
@@ -9,8 +9,8 @@
     [org.broadinstitute.firecloud-ui.common.table :refer [Table]]
     [org.broadinstitute.firecloud-ui.common.table-utils :refer [default-render]]
     [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
-    [org.broadinstitute.firecloud-ui.utils :as utils]
-    ))
+    [org.broadinstitute.firecloud-ui.utils :as utils]))
```

Comment:
FYI: The reason we leave this is so you can add a new import without seeing a two-line change in git.